packages/flutter_tools/lib/src/localizations/localizations_utils.dart

```diff
@@ -294,7 +294,14 @@ String generateString(String value) {
 /// Given a list of strings, placeholders, or helper function calls, concatenate
```

Comment:
Can `expressions` really include "helper function calls"?

Reply:
Ah you're right, I think that was an outdated comment from when I used to directly embed helper functions to generate some expressions.